pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -113,10 +125,14 @@ stdenv.mkDerivation ({
   enableParallelBuilding = parallelBuild;
 
   postPatch = ''
-    patchShebangs make
+    substituteInPlace lib/wx/c_src/wxe_impl.cpp \
+      --replace 'temp > NULL' 'temp != NULL'
```

Comment:
`--replace` is deprecated

Reply:
Use `--replace-fail` instead.

Reply:
Turns out this whole step isn't necessary when the earliest supported version of Erlang doesn't have this error.